modules/apps/remote-app/remote-app-admin-web/src/main/resources/content/Language.properties

```diff
@@ -1,3 +1,4 @@
+add-remote-web-app=Add Remote Web App
```

Comment:
It looks fine to me.

However, why not `new-remote-app` as it was already defined?

Reply:
I'd say that's up to the product team. If they think a slightly different label better explains the feature, :+1: